pdf_compressor.py

```diff
@@ -60,6 +61,14 @@ def compress(input_file_path, output_file_path, power=0):
     print("Done.")
 
 
+def get_ghostscript_path():
```

Comment:
Hi @skjerns I'm using python 2.7 so I don't have shutil.which. However I did have success in using your def to utilize the os module instead on my machine to return the correct path [as explained here](https://stackoverflow.com/questions/9877462/is-there-a-python-equivalent-to-the-which-command). I did have to add .exe to the variable though.